sysid-projects/drive/src/main/cpp/Robot.cpp

```diff
@@ -169,10 +158,7 @@ Robot::Robot() : frc::TimedRobot(5_ms) {
         // } else if (gyroType == "Romi") {
         //   m_gyro = std::make_unique<frc::RomiGyro>();
       } else {
-        std::stringstream ss(gyroCtor);
-        int port = 0;
-        ss >> port;
-        m_gyro = std::make_unique<frc::AnalogGyro>(port);
+        m_gyro = std::make_unique<frc::AnalogGyro>(std::stoi(gyroCtor));
```

Comment:
Testing the NavX would lead to this code running. Maybe stoi crashes when the constructor is not an int whereas stringstream doesn't crash?

Reply:
Yep. I just added a try-catch that defaults the port number to zero. stderr prints in projects don't show in the console during builds, so there's not much we can do to report errors besides crashing.